### PR TITLE
feat: add reimbursement by card docs

### PR DESCRIPTION
**Pull request by pedroalmeida-fidel**
[Netlify Status](https://app.netlify.com/sites/fidel-website/deploys)

### CODE REVIEW

File: reimbursement.md [hunk 1]
```diff
@@ -282,6 +282,96 @@ Find the reimbursement status in the transaction `reimbursement.status` property
 
 - `failed`: scheme request failed and `transaction.reimbursement.error` object is created. Retry is possible. [See error list for more information](https://fidel.uk/docs/reimbursement#errors).
 
+## Reimbursement by card
+
+At this point reimbursements could only be performed for a specific transaction. There are other scenarios when you just want to reimburse a card, without a transaction. In this case, you'd need to find a transaction (that belongs to the card) and is  eligible for reimbursement manually, then reimburse. Depending on your use case, this can be a time consuming and tedious process.
```
Comment (krisztina-nagy) [inline reimbursement.md:287]: I would rephrase the first two paragraphs. Users usually don't read documentation pages from the beginning to the end, so this page is actually a compilation of smaller texts that the reader will read when they think it's a solution for their question (and they usually skip the rest).

Suggestion: 
"In some cases, you may want to reimburse a card (and not a specific transaction). However, as card networks allow reimbursements only on specific transactions, the reimbursement request needs to be based on a transaction. The `/cards/{cardId}/reimbursement` endpoint helps you to find such a transaction for this case. You can simply define the `cardId`, and Fidel API will search for the most suitable transaction, and apply the reimbursement on that one.

File: reimbursement.md [hunk 2]
```diff
@@ -282,6 +282,96 @@ Find the reimbursement status in the transaction `reimbursement.status` property
...
+```json
+fileName: request.json
+{
+  "amount": 5,
+  "currency": "USD",
+  "description": "Earned Stars", //optional
+  "brandId": "518c746f-fbf4-420a-8d37-c591adc39684" // optional
+}
+```
+
```
Comment (krisztina-nagy) [inline reimbursement.md:301]: I would add a lead-in sentence here, like:
"The response contains the ..."

**Review by krisztina-nagy — APPROVED** [review]

**Review by janos-fidel — APPROVED** [review]

**Review by matheuseabra — APPROVED** [review]
